IDE_Board_Manager/prusa3dboards/platform.txt

```diff
@@ -25,14 +25,14 @@ compiler.c.elf.flags=-Os -g -flto -Wl,--gc-sections
 compiler.c.elf.cmd=avr-gcc
 compiler.S.flags=-c -g -x assembler-with-cpp -flto -MMD
 compiler.cpp.cmd=avr-g++
-compiler.cpp.flags=-c -g -Os {compiler.warning_flags} -std=gnu++11 -ffunction-sections -fdata-sections -MMD -flto -fno-fat-lto-objects -fno-exceptions -fno-threadsafe-statics -fno-rtti
+compiler.cpp.flags=-c -g -Os {compiler.warning_flags} -std=gnu++17 -ffunction-sections -fdata-sections -MMD -flto -fno-fat-lto-objects -fno-exceptions -fno-threadsafe-statics -fno-rtti
 compiler.ar.cmd=avr-gcc-ar
 compiler.ar.flags=rcs
 compiler.objcopy.cmd=avr-objcopy
 compiler.objcopy.eep.flags=-O ihex -j .eeprom --set-section-flags=.eeprom=alloc,load --no-change-warnings --change-section-lma .eeprom=0
 compiler.elf2hex.flags=-O ihex -R .eeprom
 compiler.elf2hex.cmd=avr-objcopy
-compiler.ldflags=
+compiler.ldflags=-flto -fwhole-program
```

Comment:
Just use -flto here. -fwhole-program and -flto perform the same duty, but work with different principles. -fwhole-program would require to compile and link in a single pass, which is not technically possible using arduino-builder anyway.

We are already using -flto in the compilation flags, so everything looks correct to me.

Reply:
@DRracer Please comment on that.

Reply:
ok, I agree, let's keep just -flto in ldflags